css/filetypes.scss

```diff
@@ -13,3 +13,12 @@
 .icon-filetype-draw {
 	background-image: url('../img/x-office-drawing.svg');
 }
+
+.icon-richdocuments {
+	background-image: url(../img/app-dark.svg);
+	filter: var(--background-invert-if-dark);
+}
+
+body.dark .icon-richdocuments {
```

Comment:
Let's drop it since we have one branch per major Nextcloud version ;)

Reply:
Done! We will add it manually for the backports?